compile_stdlib.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+"""
+Convert ouroborous modules to JavaScripted pyc files that we can load.
+"""
+
+# TODO: we should just import ourborous and generate the pycs from there
+
+import base64
+import glob
+import os
+import os.path
+import py_compile
+import sys
+import tempfile
+
+enabled_modules = [
+    'colorsys',
+    'copyreg',
+    'this',
+]
```

Comment:
Ideally this list would be configurable. I'm wondering if it might be desirable to use this script as the general purpose "make me a .min.js" entry point allowing end users to craft a customised batavia.min.js, with the default behavior being to build the entire system.